### DISCUSSION TITLE
Error dialog cuts of text and is not resizeable

### DISCUSSION THREAD

**Discussion by frankbenoit** [Genuitec/Copilot4Eclipse-community#57] [Help]
See this image:
![image](https://github.com/user-attachments/assets/bee763c0-c566-4f38-bdc9-074c5310a6d6)

Would be helpful to have the full text readable.

**Comment by wayneparrott** [comment]
Thx for sharing this issue. We will improve this dialog for longer error messages in the next update.

Wayne
Copilot4Eclipse Team
